scalafix/rules/src/main/scala/rsc/rules/RscCompat.scala

```diff
@@ -55,7 +55,7 @@ case class RscCompat(legacyIndex: SemanticdbIndex, config: RscCompatConfig)
           val env1 = predefScope :: scalaScope :: javaLangScope :: rootScope :: env
           stats.foldLeft(env1)(loop)
         case Import(importers) =>
-          importers.foldLeft(env)(loop)
+          return importers.foldLeft(env)(loop)
```

Comment:
Neat! After having hacked up the prototype, I knew that there was a bug somewhere, but I hadn't time to debug. Thanks for taking care of that.